sources/Load.php

```diff
@@ -1020,7 +1020,7 @@ function loadMemberContext($user, $display_custom_fields = false)
 	// Set things up to be used before hand.
 	$profile['signature'] = str_replace(array("\n", "\r"), array('<br />', ''), $profile['signature']);
 	$profile['signature'] = $parsers->parseSignature($profile['signature'], true);
-	$profile['is_online'] = (!empty($profile['show_online']) || allowedTo('moderate_forum')) && $profile['is_online'] > 0;
+	$profile['is_online'] = !empty($profile['show_online']) || (allowedTo('moderate_forum') && $profile['is_online'] > 0);
```

Comment:
Actually that's the wrong grouping. Ack.

And not exactly, I saw the useless parens while trying to figure out what was going on with it.